cdap-examples/Sports/bin/upload-results.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
```

Comment:
Can we use the CLI instead of the one off script? We can have instructions in  README of how to do this using CLI. That way we don't need a script for windows as well.